Directory.Packages.props

```diff
@@ -107,7 +107,7 @@
     <PackageVersion Include="Polly" Version="8.6.0" />
     <PackageVersion Include="RabbitMQ.Client" Version="7.1.2" />
     <PackageVersion Include="Raven.CodeAnalysis" Version="1.0.11" />
-    <PackageVersion Include="Raven.Pal" Version="7.0.841" />
+    <PackageVersion Include="Raven.Pal" Version="8.0.1" />
```

Comment:
Do we want to drop:

1. libs/librvnpal/librvnpal.linux.arm.32.so
2. libs/librvnpal/librvnpal.linux.arm.64.so

?